doc/source/cd/methods/learnedkerneldrift.ipynb

```diff
@@ -81,17 +81,17 @@
     "* `data_type`: Optionally specify the data type (e.g. tabular, image or time-series). Added to metadata.\n",
     "\n",
     "\n",
-    "Additional PyTorch keyword arguments:\n",
+    "Additional PyTorch and KeOps keyword arguments:\n",
     "\n",
     "* `device`: *cuda* or *gpu* to use the GPU and *cpu* for the CPU. If the device is not specified, the detector will try to leverage the GPU if possible and otherwise fall back on CPU.\n",
     "\n",
     "* `dataloader`: Dataloader object used during training of the kernel. Defaults to `torch.utils.data.DataLoader`. The dataloader is not initialized yet, this is done during init off the detector using the `batch_size`. Custom dataloaders can be passed as well, e.g. for graph data we can use `torch_geometric.data.DataLoader`.\n",
     "\n",
-    "Additional KeOps keyword arguments:\n",
+    "* `batch_size_predict`: Batch size used for the trained drift detector predictions. Defaults to 1,000,000 for KeOps  and 1,000 for PyTorch.\n",
```

Comment:
This is not quite true in practice since the top level detector is instantiated by default with 1mn and this default propagates to PyTorch.

Reply:
Ah good point. For the default I guess we'll have to decide on a compromise for the optimal value between the `pytorch` and `keops` implementations?

Reply:
As PyTorch is likely the more common use case, might be fine to prioritise that one.